code/game/objects/structures/morgue.dm

```diff
@@ -266,17 +266,13 @@ GLOBAL_LIST_EMPTY(crematoriums)
 		update_icon()
 
 		for(var/mob/living/M in conts)
-			if (M.stat != DEAD)
+			if (M.stat == DEAD	|| M.name == "Punishing bird")	//Only kill dead people and punishing bird
+				M.death(1)
 				M.emote("scream")
-			if(user)
-				log_combat(user, M, "cremated")
-			else
-				M.log_message("was cremated", LOG_ATTACK)
-
-			M.death(1)
-			if(M) //some animals get automatically deleted on death.
 				M.ghostize()
 				qdel(M)
+			if (M.stat != DEAD)	//Don't kill dead people
```

Comment:
Why is it here if you are checking for it above?

Reply:
I was gonna put a message here, decided against it.